backend/app/tests/api/routes/test_responses.py

```
@@ -371,83 +354,51 @@ def test_responses_endpoint_with_file_search_results(
 @patch("app.api.routes.responses.get_conversation_by_ancestor_id")
-def test_responses_endpoint_with_ancestor_conversation_found(
+def test_responses_endpoint_with_ancestor_conversation_not_found(
     mock_get_conversation_by_ancestor_id,
     mock_create_conversation,
     mock_get_ancestor_id_from_response,
     mock_tracer_class,
     mock_get_assistant,
     mock_get_credential,
     mock_openai,
+    mock_process_response,
     db,
```

Ruff (0.12.2)

Unused function argument: `db`

(ARG001)